src/main/java/adris/altoclef/util/Dimension.java

```diff
@@ -1,6 +1,7 @@
 package adris.altoclef.util;
 
-public enum Dimension {
+
+public enum Dimension { // TODO: 2021-05-22 what if there are more dimensions on the server? eg. non-vanilla
```

Comment:
Hmmmmmmm I can try to remove the necessity for this (ex. for MLGBucket, check whether water evaporates in the current dimension instead of if dimension == Dimensions.NETHER) but there are some scenarios where that doesn't work.

I can also add some functions somewhere (WorldUtil?) like isInOverworld, isInNether, isInEnd and use those instead, sort of "manually" dealing with each instead of having a variable to pass. It might not make stuff all that annoying actually, good to consider.